### PR TITLE
Proofread and typo fix in heavy_tails

### PR DESCRIPTION
**Pull request by mtiley**

### CODE REVIEW

File: source/rst/heavy_tails.rst [hunk 1]
```diff
@@ -425,7 +425,7 @@ phenomena.
 
 To illustrate this fact in a simple way, let us consider an economy with
 100,000 firms, an interest rate of ``r = 0.05`` and a corporate tax rate of
-15%.
+15\%.
```
Comment (jstac) [inline source/rst/heavy_tails.rst]: Are you sure we need this @mtiley ?

It seems to render OK in the notebook without.

Which formats did you check it in?  I was wondering about this and we should document what you find for other Jupinx users (CC @mmcky).

Comment (mtiley) [reply source/rst/heavy_tails.rst]: No, just noticed it was used earlier in the lecture and went for consistency. It looked fine in the jupyter notebook but I'm having a problem with jupinx website generation at the moment - thought including it the safer option but perhaps not?

Comment (mmcky) [reply source/rst/heavy_tails.rst]: thanks @mtiley -- I think both will work. But the `\` shouldn't be needed here.

**Comment by mmcky** [comment]
as a tip you can also compile single files which makes website compilation a lot faster

```
jupinx -cw --files=source/rst/heavy_tails.rst
```

File: source/rst/heavy_tails.rst (hunk 1 above)
Comment (jstac) [reply source/rst/heavy_tails.rst]: Hi @mmcky , just to check my understanding, `\` is not needed for notebook, web page or pdf?

Is this the same for `$` and `#` displayed in text?

Is this worth documenting in the Jupinx documentation?

Comment (mmcky) [reply source/rst/heavy_tails.rst]: hey @jstac let me add a test case to `sphincontrib-jupyter` with all these cases `with` and `without` and I will then say with certainty. Then we can document. Do you remember if this was needed for `html`?

Comment (jstac) [reply source/rst/heavy_tails.rst]: Sorry @mmcky, I don't understand your last question.  We want all outputs to work, don't we?

Comment (mmcky) [reply source/rst/heavy_tails.rst]: Just referring to when we used `sphinx` to compile to `html` directly -- was wondering if these `\` were included at that point.

Comment (mmcky) [reply source/rst/heavy_tails.rst]: the following:

```rst
Syntax
=======

This contains test cases for using special syntax

If we use ``%`` in text:

15%

and with ``\%``

15\%

IF we use ``#`` in text:

#1

and with ``\#``

\#1

If we use ``$`` in text:

$100,000

and with ``\$``

\$100,000

```

produces:

image

**Pull request merged**